include/eve/traits/overload/supports.hpp

```diff
@@ -68,6 +68,35 @@ namespace eve
   template<auto Func>
   using tag_t = std::remove_cvref_t<decltype(Func)>;
 
+  //====================================================================================================================
+  //! @addtogroup extensions
```

Comment:
I appreciate what you want to do here but I don't like the word accept here - because you'd expect to accep eve options too. How about `as_simple_option` ?  Not amazing but I'm open to suggestions

Reply:
`supported_options<...>`  ?